cmd/envtool/envtool.go

```diff
@@ -179,6 +179,35 @@ func setupMongodbSecured(ctx context.Context, logger *zap.SugaredLogger) error {
 	return waitForPort(ctx, logger.Named("mongodb_secured"), 47018)
 }
 
+// setupMySQL configures `mysql` container.
+func setupMySQL(ctx context.Context, logger *zap.SugaredLogger) error {
+	if err := waitForPort(ctx, logger.Named("mysql"), 3306); err != nil {
+		return err
+	}
+
+	eval := "mysql -u root -ppassword -e \"GRANT ALL PRIVILEGES ON *.* TO 'username'@'%';\""
```

Comment:
That's okay for now, but we should use the backend code once we have it.

Let's add a comment for that.

Reply:
I'm not sure what you mean that we should use the backend code for this.

Reply:
We should do a thing similar to `setupAnyPostgres` by using `internal/backends/mysql/...` package there instead of `docker compose exec`. But that's not possible now. So I think we should add a comment about that future change.

Reply:
Ohh okay. Got it